roles/kubernetes/node/defaults/main.yml

```diff
@@ -141,6 +141,7 @@ vsphere_public_network: "{{ lookup('env', 'VSPHERE_PUBLIC_NETWORK')|default('')
 # azure_security_group_name:
 # azure_vnet_name:
 # azure_route_table_name:
+# azure_vmtype:
```

Comment:
add some help and default values in comment example

for example 
# Supported values are 'standard' or 'vmss'
# azure_vmtype: standard

Reply:
Update the roles/kubernetes/node/defaults/main.yml file with the above comment and also. Update the docs/azure.md